Sodium/Bytes.swift

```diff
@@ -0,0 +1,21 @@
+import Foundation
+
+public typealias Bytes = Array<UInt8>
+
+extension Array where Element == UInt8 {
+    init (count bytes: Int) {
+        self.init(repeating: 0, count: bytes)
+    }
+
+    public var utf8String: String? {
+        return String(data: Data(bytes: self), encoding: .utf8)
+    }
+}
+
+extension ArraySlice where Element == UInt8 {
+    var bytes: Bytes { return Bytes(self) }
+}
+
+public extension String {
+    var bytes: Bytes { return Bytes(self.utf8) }
```

Comment:
I would make it more clear that the string is UTF-8 encoded by calling this `utf8bytes`.